autoscale_cloudcafe/autoscale/client.py

```diff
@@ -940,12 +941,17 @@ def get_node_info_detail(self, pool_id, node_id):
         return self.request('GET', url,
                             response_entity_type=RackConnectLBNodeDetail)
 
-    def remove_node_from_pool(self, pool_id, node_id, requestslib_kwargs=None):
+    def remove_server_from_pool(self, pool_id, server_id,
+                                requestslib_kwargs=None):
         """
-        :summary: Remove a node from a lb pool
+        :summary: Remove a server from a lb pool using the bulk delete API
         """
-        url = self.url + '/load_balancer_pools/{0}/nodes/{1}'.format(pool_id,
```

Comment:
Mimic only implements the bulk delete/bulk add endpoint, because that's all that otter uses, so either single node delete has to be added to mimic, or the tests can delete via the same method, which seems fine.  This change also is actually in #1087, which this branch is based off of.